src/obstacle_avoidance/launch/go_through_hole.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+    <node name="my_node" pkg="sample_package" type="my_node" output="screen">
```

Comment:
type should be the executable you called it "go_through_hole", while pkg should be the package it is in which is "obstacle_avoidance".